docs/supported-providers.md

```diff
@@ -80,14 +84,42 @@ If you are using AirVPN or other provider with credentials in the config file, y
 to set `OPENVPN_USERNAME` and `OPENVPN_PASSWORD` as this is required by the startup script.
 They will not be read by the .ovpn file, so you can set them to whatever.
 
-Note that you still need to modify your .ovpn file as described in the previous section.
-If you have an separate ca.crt, client.key or client.crt file in your volume mount should be a folder containing both the ca.crt and the .ovpn config.
+If `-e OPENVPN_CONFIG=` variable has been omitted from the `docker run` command the .ovpn config file must be named default.ovpn.
+If you have an separate ca.crt, client.key or client.crt file, then refer to the section below instead.
+
 
-Mount the folder contianing all the required files instead of the openvpn.ovpn file.
-`-v /path/to/your/config/:/etc/openvpn/custom/`
+## Using a local set of .ovpn files from provider
+If you do not want the build to download config from the provider but actually load you own config locally, please follow these steps
 
-Additionally the .ovpn config should include the full path on the docker container to the ca.crt and additional files.
-`ca /etc/openvpn/custom/ca.crt`
+**Grab all files from your provider** (usually a .zip file to download & unzip)
 
-If `-e OPENVPN_CONFIG=` variable has been omitted from the `docker run` command the .ovpn config file must be named default.ovpn.
-If `-e OPENVPN_CONFIG=` is used with the custom provider the .ovpn config and variable must match as described above.
+**Copy them in a repository**, there should be .ovpn files and a ca.cert as well (example below /volume1/docker/transmission/ipvanish/)
+
+**Mount the volume**
+Compose sample:
+```
+             - /volume1/docker/transmission/ipvanish/:/etc/openvpn/custom/
+```
+**Declare the Custom provider, the target server and login/password**
+Compose sample:
+```
+            - OPENVPN_PROVIDER=custom
+            - OPENVPN_CONFIG=ipvanish-UK-Maidenhead-lhr-c02
+            - OPENVPN_USERNAME=uourlogin
```

Comment:
yourlogin..
although I dislike conjoints like this..
I think either use "user/pass" as elsewhere or <username>/<password>

Reply:
I'm completely ok with that change, I  don't see how to update the wiki though (very noob to Github as you may notice)